Samples/PublicSamples/EchoBot/src/EchoBot/appsettings.json

```diff
@@ -1,20 +1,16 @@
-﻿{
-  "AzureSettings": {
-    "AadAppId": "",
-    "AadAppSecret": "",
-    "ServiceDnsName": "bot.example.com",
-    "CertificateThumbprint": "",
-    "BotCallingInternalPort": "9442", // join calls from LB to this local port 443 => 9442
```

Comment:
Could you help point where we have defining these now? I might have missed this in the new setup

Reply:
All settings are pulled in from environment variables. If running in dev, you can use a .env file to add the secrets and configs. I'll add a .env template file.